webpack.extension.dev.js

```diff
@@ -77,6 +77,7 @@ module.exports = [
                 'process.env.ATLASCODE_FX3_ENVIRONMENT': JSON.stringify(process.env.ATLASCODE_FX3_ENVIRONMENT),
                 'process.env.ATLASCODE_FX3_TARGET_APP': JSON.stringify(process.env.ATLASCODE_FX3_TARGET_APP),
                 'process.env.ATLASCODE_FX3_TIMEOUT': JSON.stringify(process.env.ATLASCODE_FX3_TIMEOUT),
+                'process.env.CI': JSON.stringify(process.env.CI),
```

Comment:
Apparently some transitive dependencies have special behaviour depending on `$CI`. We probably will too, eventually :D